.travis.yml

```diff
@@ -28,7 +30,7 @@ branches:
 before_script:
   # Install deps
   - composer update $COMPOSER_FLAGS --dev --prefer-source --no-interaction
-  - sh -c "if [ '$TRAVIS_PHP_VERSION' = '7' ]; then wget https://phar.phpunit.de/phpunit-5.7.phar; fi"
+  - sh -c "if [ '$TRAVIS_PHP_VERSION' = '7.0' ] || [ '$TRAVIS_PHP_VERSION' = '7.1' ] || [ '$TRAVIS_PHP_VERSION' = '7.2' ]; then wget https://phar.phpunit.de/phpunit-5.7.phar; fi"
```

Comment:
lets try with bash: ``bash -c "if [[ '$TRAVIS_PHP_VERSION' == '7.'* ]] ; then wget https://phar.phpunit.de/phpunit-5.7.phar; fi" ``

and yes, bash is a really messed up thing :-/. the `*` is outside the quotes...